.github/approve_config.yml

```diff
@@ -16,7 +16,8 @@ approvals:
         - tchaton
         - williamFalcon
     - name: 'Lightning Apps'
-      minimum: 0
+      minimum: 1
```

Comment:
Is this intended? It means that both groups will need to approve each PR

Reply:
Seems yes, if I tried 0, it expected that everyone need to review...